src/main/resources/db/createTables.sql

```diff
@@ -0,0 +1,85 @@
+-- Table: User
+CREATE TABLE IF NOT EXISTS Users (
+    id SERIAL PRIMARY KEY,
+    username VARCHAR(50) NOT NULL UNIQUE,
+    email VARCHAR(100) NOT NULL UNIQUE
+);
+
+CREATE TYPE problem_level AS ENUM ('ok', 'failure');
+
+-- Table: Problems
+CREATE TABLE IF NOT EXISTS Problems (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    description TEXT NOT NULL,
+    level problem_level NOT NULL,
+    test_cases JSONB NOT NULL
+);
+
+-- Table: Languages
+CREATE TABLE IF NOT EXISTS Languages (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50) NOT NULL UNIQUE
+);
+
+
+CREATE TYPE status AS ENUM ('ok', 'failure');
+
+
+-- Table: Submissions
+CREATE TABLE IF NOT EXISTS Submissions (
+    id UUID PRIMARY KEY,
+    problem_id INT NOT NULL,
+    language_id INT NOT NULL,
+    timestamp TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP,
```

Comment:
```suggestion
    timestamp TIMESTAMPTZ(0) DEFAULT CURRENT_TIMESTAMP,
```
explicitly set 0 precision